### DISCUSSION TITLE
Open Collective!

### DISCUSSION THREAD

**Discussion by staltz** [nodejs-mobile/nodejs-mobile#42] [Announcements]
A big announcement! Many of you have requested more maintenance for Node.js Mobile, and I have found a solution for this in the form of donation-driven work. We now have an [**Open Collective page**](https://opencollective.com/nodejs-mobile) to gather donations. With your contributions, I want to keep this project alive and thriving, starting with a few immediate issues at hand:

- Upgrading to LTS v18.x
- (React Native) Fixing warnings and errors with newer versions of React Native
- Supporting various flavors of iOS (simulator on M1, simulator on Intel, etc)
- [Supporting at least ICU small](https://github.com/nodejs-mobile/nodejs-mobile/issues/20)
- [Fix Android native modules redundantly rebuilt](https://github.com/nodejs-mobile/nodejs-mobile-react-native/issues/45)
- Relaunch the documentation site
- Reviewing and merging PRs more frequently

For Cordova support, we need another dedicated maintainer for that. But for now the packages `nodejs-mobile` and `nodejs-mobile-react-native` are going to be well maintained.

If you can donate some amount, thank you!

**Comment by holmesworcester** [comment]
Great news! We're currently using Node.js Mobile for [Quiet](https://github.com/TryQuiet/quiet/) and will definitely be supporting this work!
